twopence/run

```diff
@@ -355,7 +355,7 @@ def nfs_show_kernel_exports():
 # the exports file, and use exportfs to provide the reader with
 # hints of what acutally *did* get exported.
 ##################################################################
-def nfs_write_exports(server, exports):
+def nfs_write_exports(server, exports, reload_nfs = "YES"):
```

Comment:
Hi Dario,

On Thu, 2016-06-02 at 06:50 -0700, Dario Maiocchi wrote:
> In twopence/run:
> 
> > @@ -355,7 +355,7 @@ def nfs_show_kernel_exports():
> >  # the exports file, and use exportfs to provide the reader with
> >  # hints of what acutally *did* get exported.
> >  ##################################################################
> > -def nfs_write_exports(server, exports):
> > +def nfs_write_exports(server, exports, reload_nfs = "YES"):

Can we actually use booleans for this? Something like the attached...
that patch is untested, and maybe it needs an explicit journal.success()
when bailing out of nfs_write_exports early - but you get the idea.

Olaf


-- 
Olaf Kirch - VP SUSE Linux Enterprise; R&D (okir@suse.com)
SUSE Linux GmbH, Maxfeldstr. 5, 90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Dilip Upmanyu, Graham Norton,
HRB 21284 (AG Nürnberg)